Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -598,7 +598,25 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
                 innerElement.append(item);
         }
 
-        let swatch = new WI.InlineSwatch(type, valueObject, {readOnly: !this._isEditable()});
+        let colorCSSVariables = this.property.ownerStyle.nodeStyles.computedStyle.variablesForType(WI.CSSStyleDeclaration.VariablesGroupType.Colors);
```

Comment:
i think we need to do some sort of deduplication here as it's possible for multiple variables to have the same color value.  or are we OK with having multiple `WI.InlineSwatch` displaying the same color but with different variables?